### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by claudiokerekes**
#### Board:
* #1 
---
#### Description:
* Check [readme file](https://github.com/rootstrap/poc-action-mailbox/tree/initial-setup#readme)
---
#### Notes:
* We will add comments to help the Rootstrap team better understand
---

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,57 @@
...
+
+## Working with Action Mailbox in development 
+It's helpful to be able to test incoming emails in development without actually sending and receiving 
+real emails. To accomplish this, there's a conductor controller mounted at
+[rails/conductor/action_mailbox/inbound_emails](localhost:3000/rails/conductor/action_mailbox/inbound_emails), which gives you an index of all the InboundEmails 
+in the system, their state of processing, and a form to create a new InboundEmail as well.
+[doc](https://guides.rubyonrails.org/action_mailbox_basics.html#working-with-action-mailbox-in-development)
+
+**NOTE** 
+To use this feature, you application must not be api only. You can set `config.api_only = false` into config/application.rb file
```
Comment (megatux) [inline README.md:36]: api -> API

File: docker-compose.yml [hunk 2]
```diff
@@ -25,8 +25,6 @@ services:
       POSTGRES_USER: ${POSTGRES_USER-postgres}
       POSTGRES_PASSWORD: ${POSTGRES_PASSWORD-postgres}
     volumes:
-      - db_data:/var/lib/postgresql/data
+      - ~/rootstrap/projects/databases/action-mailbox:/var/lib/postgresql/data
```
Comment (megatux) [inline docker-compose.yml:28]: 🤔 this path seems a little bit custom, maybe something more generic like the previous one?

**Review by megatux — COMMENTED** [review]
LGTM :+1: Left a comment on a path in the Docker volume setup.

File: config/application.rb [hunk 3]
```diff
@@ -59,7 +59,7 @@ class Application < Rails::Application
     # Only loads a smaller set of middleware suitable for API only apps.
     # Middleware like session, flash, cookies can be added back manually.
     # Skip views, helpers and assets when generating a new resource.
-    config.api_only = true
+    config.api_only = false
```
[the remark this answers is not in the document]
Comment (claudiokerekes) [reply config/application.rb:62]: It is already added. :)

[the remark this answers is not in the document]
Comment (megatux) [reply config/application.rb:62]: how much of the non API-only mode will the app use? Is still valid to fork from the rails-api-base and not from a normal "rails-base" app? (we [had a Rails-base](https://github.com/rootstrap/rails_base) project some time ago, but it is now deprecated)

File: docker-compose.yml (hunk 2 above)
Comment (claudiokerekes) [reply docker-compose.yml]: done @megatux

File: config/application.rb (hunk 3 above)
[the remark this answers is not in the document]
Comment (claudiokerekes) [reply config/application.rb:62]: I understand your point but I don't know if you need to. The plan is not to use this repo in production. It's just knowing how Action Mailbox works. WDYT??

[the remark this answers is not in the document]
Comment (megatux) [reply config/application.rb:62]: sure, makes sense, let's see how things turn out

**Review by megatux — COMMENTED** [review]
LGTM :+1:!!!

**Review by megatux — APPROVED** [review]
LGTM :+1:!!!

**Pull request merged**